homeassistant/components/screenlogic/entity.py

```diff
@@ -50,10 +54,12 @@ def __init__(
         """Initialize of the entity."""
         super().__init__(coordinator)
         self.entity_description = entity_description
-        self._data_path = self.entity_description.data_path
-        self._data_key = self._data_path[-1]
-        self._attr_unique_id = f"{self.mac}_{self.entity_description.key}"
+        self._data_root = self.entity_description.data_root
```

Comment:
It doesn't look like `self._data_root` is used anywhere outside this method. It doesn't need to be an instance attribute then.

Reply:
[c7a8c41](https://github.com/home-assistant/core/pull/100173/commits/c7a8c41b964a06a32c33c6cc32565c32d669a2af)